### PR TITLE
Added the fingbox homepage takeover

### LINKED ISSUE #4088: Fingbox takeover - build
https://docs.google.com/document/d/1P8BC483zp5bYS5SuKimBHSIBlLhyEL9O_AcH3-pAjVc/edit#

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

Added the fingbox homepage takeover

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare the text and link to the [copy doc](https://docs.google.com/document/d/1Af5L4jNaEj0U4KOllcTCBHfLbqBXfbzY7AsUg_X9Rqo/edit?ts=5baa4881#) and the design to the issue #4088
    - you should see 3 takeovers, fing, ai, multicloud

## Issue / Card

Fixes #4088

## Screenshots

large

image

small

image

### CODE REVIEW

File: templates/takeovers/_fing_webinar.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<section data-lang="en" lang="en-gb" class="p-strip is-deep p-takeover--fing-webinar js-takeover {% if  default %}u-hide{% endif %}" {% if default %}data-default="true"{% endif %}>
```
Comment (anthonydillon) [inline templates/takeovers/_fing_webinar.html]: If the background is dark you should add `is-dark` class which will adjust so the text to white.

File: templates/takeovers/_fing_webinar.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+<style>
+  .p-takeover--fing-webinar {
+    background-color: #006BA1;
+    background-image:  linear-gradient(32deg, #006ba1 0%,#009eee 84%,#08acff 100%);
+  }
+
+  .p-takeover--fing-webinar .p-takeover__title {
+    font-weight: 100;
+    color: #fff;
```
Comment (anthonydillon) [inline templates/takeovers/_fing_webinar.html]: You shouldn't need this with `is-dark` on the strip.

File: templates/takeovers/_fing_webinar.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  }
+
+  .p-takeover--fing-webinar .p-takeover__text {
+    color: #fff;
```
Comment (anthonydillon) [inline templates/takeovers/_fing_webinar.html]: You shouldn't need this with `is-dark` on the strip.

File: templates/takeovers/_fing_webinar.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    .p-takeover--fing-webinar {
+
+      background-image: url('{{ ASSET_SERVER_URL }}5175ed7b-fingbox-left-overlay.svg'), url('{{ ASSET_SERVER_URL }}1ce949ac-fingbox-right-overlay.svg'), linear-gradient(32deg, #006ba1 0%,#009eee 84%,#08acff 100%);
+      background-repeat: no-repeat;
+      background-size: contain;
+      background-position: bottom left, bottom right;
+    }
+  }
+
+  @keyframes bounce {
```
Comment (anthonydillon) [inline templates/takeovers/_fing_webinar.html]: I can see where you are using this animation.

**Comment by anthonydillon** [comment]
The latest copy doesn't make the takeover look like the design: 
screenshot_20181004-121146__01

File: templates/takeovers/_fing_webinar.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
+<section data-lang="en" lang="en-gb" class="p-strip is-dark is-deep p-takeover--fing-webinar js-takeover {% if  default %}u-hide{% endif %}" {% if default %}data-default="true"{% endif %}>
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1 class="p-takeover__title">Future-proofing Fingbox the home network monitoring device</h1>
+      <p class="p-takeover__text">Learn how Fing uses Ubuntu Core and Snaps to speed development, save budget and resources and ensure a future-proofed product.</p>
+      <img class="p-takeover__image u-hide--medium u-hide--large" alt="Fing's Fingbox" src="{{ ASSET_SERVER_URL }}3811386c-fingbox_w-shadow.png" />
+      <p class="p-button--neutral"><a href="https://www.brighttalk.com/webcast/6793/336519" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'fing webinar takeover', 'eventLabel' : 'Future-proofing Fingbox the IoT home network monitoring device', 'eventValue' : undefined });" style="color: #000;">Register for the webinar</a></p>
```
Comment (caldav) [inline templates/takeovers/_fing_webinar.html]: Landing page (#4110) : `/engage/future-proof-iot?utm_medium=takeover`

Comment (caldav) [reply templates/takeovers/_fing_webinar.html]: (And no need for the manual GA event)

**Comment by kwm14** [comment]
## Design review
Looking good @pmahnke animation is on point 💯 just one update to be made on takeover heading
- [ ] missing 'IoT' in the heading 'Future-proofing Fingbox the IoT home network monitoring device'

File: templates/takeovers/_fing_webinar.html [hunk 6]
```diff
@@ -0,0 +1,57 @@
+<section data-lang="en" lang="en-gb" class="p-strip--dark is-deep p-takeover--fing-webinar js-takeover {% if  default %}u-hide{% endif %}" {% if default %}data-default="true"{% endif %}>
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1 class="p-takeover__title">Future-proofing Fingbox the home network monitoring device</h1>
+      <p class="p-takeover__text">Learn how Fing uses Ubuntu Core and Snaps to speed development, save budget and resources and ensure a future-proofed product.</p>
+      <img class="p-takeover__image-small u-hide--medium u-hide--large" alt="Fing's Fingbox" src="{{ ASSET_SERVER_URL }}3811386c-fingbox_w-shadow.png" />
+      <p class="p-button--neutral"><a href="/engage/future-proof-iot" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'fing webinar takeover', 'eventLabel' : 'Future-proofing Fingbox the IoT home network monitoring device', 'eventValue' : undefined });" style="color: #000;">Register for the webinar</a></p>
```
Comment (caldav) [inline templates/takeovers/_fing_webinar.html]: UTMs are missing, should be /engage/future-proof-iot?utm_medium=takeover&utm_campaign=FY19_IOT_WBN_Fing

File: templates/takeovers/_fing_webinar.html (hunk 1 above)
Comment (pmahnke) [reply templates/takeovers/_fing_webinar.html]: it was .p-strip--dark

**Comment by kwm14** [comment]
LGTM 👍

**Pull request merged**